actionpack/lib/action_dispatch/system_test_case.rb

```diff
@@ -151,8 +151,10 @@ def self.start_application # :nodoc:
     #   driven_by :selenium, using: :firefox
     #
     #   driven_by :selenium, using: :headless_firefox
-    def self.driven_by(driver, using: :chrome, screen_size: [1400, 1400], options: {}, &capabilities)
-      driver_options = { using: using, screen_size: screen_size, options: options }
+    #
+    #   driven_by :selenium, name: :custom_driver_name
+    def self.driven_by(driver, using: :chrome, screen_size: [1400, 1400], options: {}, name: nil, &capabilities)
```

Comment:
I think that `name` will not be an often used kwarg, so I'm thinking maybe it should go into the options hash instead and then deleted from the hash before being sent to capybara.